components/Cards/README.md

```diff
@@ -252,3 +255,92 @@ id<MDCColorScheming> colorScheme = [[MDCSemanticColorScheme alloc] init];
 ```
 <!--</div>-->
 
+
+## Accessibility
+
+<!-- Extracted from docs/accessibility.md -->
+
+### MDCCard Accessibility
+
+To help ensure your cards are accessible to as many users as possible, please be sure to review the following 
+recommendations:
+
+#### Accessibility for Cards in a Collection
+
+Since assistive technologies visit all cards in a collection in a sequential order, it is often easier to distinguish 
+between elements that belong to different cards, by aggregating all the card's information to a single sentence. 
+This can be done by setting an appropriate [`accessibility
+Label`](https://developer.apple.com/documentation/uikit/uiaccessibilityelement/1619577-accessibilitylabel) for the 
+card. In addition, the card's [`isAccessiblity
+Element`](https://developer.apple.com/documentation/objectivec/nsobject/1615141-isaccessibilityelement) 
+property should be enabled, so it is treated as a single entity by assistive technologies.
```

Comment:
copyedit alternative: Cards are a container element and isAccessibiltElement turns off individually selecting its subelements.